src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
@batkinson Since it turns out JR printing isn't what's causing the bulk of the problems on the Briefcase side, maybe it isn't as much of a priority? @dcbriccetti would you like to "do it right"? I filed issue #152.